subprojects/emcee/worker/src/test/kotlin/com/avito/emcee/worker/rest/HttpServerTest.kt

```diff
@@ -20,15 +20,24 @@ import java.net.ServerSocket
 @ExperimentalCoroutinesApi
 class HttpServerTest {
 
-    private var port: Int? = null
+    private companion object {
+        private fun findAvailablePort(): Int {
+            val s = ServerSocket(0)
+            val port = s.localPort
+            s.close()
+            return port
+        }
+
+        private val port: Int = findAvailablePort()
```

Comment:
It should be found on each call, for each test. Not only once.

Reply:
Before in test was @BeforeAll annotation I think that means that we make it one time before tests
What the point each time find availablePorts